tests/Feature/UuidTest.php

```diff
@@ -4,6 +4,7 @@
 
 use Tests\Fixtures\Post;
 use Tests\Fixtures\UncastPost;
+use Tests\Fixtures\CustomCastUuidPost;
```

Comment:
Is this a new file you created but forgot to add in the commit?

Reply:
Sheesh. Yes, it is. I will resubmit. THANK YOU!

Reply:
FYI; you do not normally need to make a new PR if you have more commits to push on the same branch. Only if you need to throw away the branch, make big squashing changes or force-push changes is a new PR needed, methinks. (I'm a fan of PR'ing early and pushing up new commits about a dozen times before I'm happy enough to merge, myself.)

Reply:
Thanks for that. I very rarely make PR to any repo's that aren't owned by me. I'll look into that so I can be a better community member for next time. Thanks for letting me know there's a better way. Much appreciated!!